src/Constants.py

```diff
@@ -1,5 +1,11 @@
 from enum import Enum
 
+# Persistent data directories
+BASE_DIR = "../trd-data"
```

Comment:
Not sure why you want to go one folder back to define the base directory? Seems very hacky ^^

Reply:
just not in the git folder but one above? Is it hacky? What's your idea?

Reply:
not sure why you are in the git folder in the first place? maybe could let the user decide where he wants the base directory or just use the repo as base.

Reply:
I changed some default values and I use the ~ again. It turns out that we needed to play a litte with the different os.path functions to get it properly returning correct paths for windows. The way I do it now is not breaking backward compatibility IMO